### ISSUE TITLE
Console Multisessions: Keyboard Accessibility / Mouse Control Issues

### ISSUE THREAD

**Issue by midleman** [posit-dev/positron#6451]
## Describe the issue:
I'm unable to use keyboard to tab through all the metadata dialog options or the console tab list.

## Steps to reproduce the issue:

1. Verify can use keyboard to navigate through all buttons on Console toolbar - PASS
32 Display metadata dialog using keyboard (press Enter) - PASS
3. Verify can tab through all menu options - FAIL
4. Verify you can use keyboard navigation to change the active console session from console tab list - FAIL
5. Verify you can select all/copy/paste into the input field when renaming a session - FAIL
6. Verify you can use mouse to highlight/select text in the input field when renaming a session -FAIL

### Notes

See https://www.w3.org/WAI/ARIA/apg/patterns/tabs/ for keyboard accessibility pattern for console tab list. Also consider work done for https://github.com/posit-dev/positron/pull/1896

**Comment by midleman** [comment]
### Verified Fixed
Positron Version(s) : [2025.06.0-124](https://github.com/posit-dev/positron-builds/releases/tag/2025.06.0-124)
OS Version(s) : MacOS

### Test scenario(s) 
Confirmed accessibility is working as expected.

### Link(s) to test cases run or created: 
Added a basic test: https://github.com/posit-dev/positron/blob/98faeb04e20b9d16df9c56e732b95c1d40f69616/test/e2e/tests/sessions/session-mgmt.test.ts#L175-L200
